rtl/eclrtl/rtlnewkey.cpp

```diff
@@ -370,9 +370,9 @@ class ValueTransition : implements CInterfaceOf<IValueTransition>
             // x >= 'ab' becomes x > 'a' => clear the equal item
             // x < 'ab' becomes x <= 'a' => set the equal item
             if (newMask & CMPlt)
-                newMask &= ~CMPeq;
```

Comment:
This seems like a pretty significant change - was it completely broken before? If so why was that not noticed?

Reply:
Yes it was broken before.  It was because the unit tests where checking consistency, but didn't cover this case.  I'll see if I can add an explicit test.